libs/net/elastic/TNOElasticClient.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Text.Json;
+using Elasticsearch.Net;
+using Microsoft.Extensions.Options;
+using Nest;
+using TNO.Core.Exceptions;
+using TNO.Core.Extensions;
+using TNO.Core.Http;
+using TNO.Elastic.Models;
+
+namespace TNO.Elastic
+{
+    /// <summary>
+    /// The TNOElasticClient class
+    /// </summary>
+    public class TNOElasticClient : ElasticClient, ITNOElasticClient
```

Comment:
Eventually we'll get rid of the NEST `ElasticClient` entirely.  For now we're making it so we can do both.